src/modules/transactions/components/Table.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <table class="table-transactions">
+        <tr>
+          <th>Título</th>
+          <th>Descrição</th>
+          <th>Status</th>
+          <th>Valor</th>
+          <th>Visualizar</th>
+        </tr>
+        <tr v-for="transaction in transactions" :key="transaction.id">
+          <td>{{transaction.title}}</td>
+          <td>{{transaction.description}}</td>
+          <td>{{transaction.status}}</td>
+          <td>{{transaction.amount | moneyFormat}}</td>
+          <td><Button @click.native="openModalTransaction(transaction.id)" /></td>
```

Comment:
@vitor-esser Uma dúvida, foi necessário usar o '.native' nesse evento de click ?

Reply:
@vitor-esser Ta certo, notei agora que foi por que o Button é um componente

Reply:
@AndersonFSP que eu saiba sim, pois o Button é um componente e então vou clicar nele para abrir o modal.

Reply:
Complementando o comentário, não é certo gerenciarmos um componente dessa forma usando Click.Native. O correto é passarmos uma função via @Prop para ser executada no @Click do button ou esperarmos um evento(this.$emit) vindo do filho(Button) e executarmos alguma ação a partir disso

Reply:
[048a2c3](https://github.com/vitor-esser/vitoresser-desafio-vue/pull/1/commits/048a2c34b7ce61c2d59e02a4d8a61762622d75d6)